addons/mail/models/ir_attachment.py

```diff
@@ -95,6 +97,11 @@ def _to_store_defaults(self, target):
             Store.One("thread", [], as_thread=True),
             "type",
             "url",
+            Store.Attr(
+                "thumbnail",
+                lambda a: f"data:image/jpeg;base64,{a.thumbnail.decode()}",
```

Comment:
Now it's an open question: do we really want to send the thumbnail here, rather just its existence?
And let the image itself be fetched through standard image routes + browser cache?

Reply:
This would solve a lot of issue with base64 stuff! I'll work on that.